tests/unit_tests/tools/test_tool_repository.py

```diff
@@ -191,16 +231,20 @@ def test_default_intel_suite(category, fake_process: FakeProcess) -> None:
     """
     Tests setting default suite to "classic-intel" produces correct tools.
     """
-    fake_process.register(['icc', '-V'], stdout='icc (ICC) 1.2.3 foo')
-    fake_process.register(['ifort', '-V'], stdout='ifort (IFORT) 1.2.3 foo')
+    fake_process.register(['icc', '--version'], stdout='icc (ICC) 1.2.3 foo')
+    fake_process.register(['ifort', '--version'],
+                          stdout='ifort (IFORT) 1.2.3 foo')
```

Comment:
It's getting increasingly hard to find information on the `ifort`/`icc` compilers but I did some checking and `-V` is the correct argument for getting what Intel call "the logo" which is the version information.

What was the reason for changing this?

Reply:
The main reason for this change is that `-V` doesn't work (our Intel builds stopped working). Yes, the compiler will report the version number, but it will not be matched with the regex used by Fab for ifort/icc, so the compilers will be considered not available. The `--version` command line option is documented in the `--help` output of the compilers:
```
--version
          display GCC style version information
```
and I've just now tested it with 16 different old intel compilers from 2019.3.199 to 2021.10.0.

Am happy for you to change the regex so that it works of course.